featuretools/tests/feature_function_tests/test_primitive_base.py

```diff
@@ -96,3 +96,7 @@ def test_return_type_inference_direct_feature(es):
 def test_return_type_inference_time_index(es):
     last = Last(es["log"]["datetime"], es["customers"])
     assert last.variable_type == Datetime
+
+    nti = NumericTimeIndex("datetime", es["log"])  # create a numeric time index variable
```

Comment:
maybe a little cleaner to make a second test using the _with_integer_time_index_ option for `make_ecommerce_entityset` instead

Reply:
done